public/app/features/query/utils.ts

```diff
@@ -0,0 +1,3 @@
+import { createMonitoringLogger } from '@grafana/runtime';
+
+export const queryLogger = createMonitoringLogger('features/query');
```

Comment:
```suggestion
export const queryLogger = createMonitoringLogger('features.query');
```
Should we use `.` to be consistent with the rest?

Reply:
Good catch! Fixed in d9258d12509c6b21bbd47451914e6b65745b5b88